packages/amplify-cli/src/__tests__/extensions/amplify-helpers/remove-resource.test.ts

```diff
@@ -215,7 +215,7 @@ describe('remove-resource', () => {
           },
         },
       });
-      expect(context.filesystem.remove).toBeCalledWith('backendDirPath/function/lambda1');
+      expect(context.filesystem.remove).toBeCalledWith(path.join("backendDirPath", "function","lambda1"));
```

Comment:
Please use single quotes and add a space after the last comma.

Reply:
Committed these changes